compat/xaccept.cc

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "squid.h"
+#include "compat/xaccept.h"
+
+#if _SQUID_WINDOWS_ || _SQUID_MINGW_
+#if HAVE_WINDOWS_H
+#include <windows.h>
+#endif
+#if HAVE_WKINSOCK2_H
+#include <winsock2.h>
+#endif
```

Comment:
Do not include these `windows.h` and `winsock2.h` here. For same reasons as already discussed in earlier PR these includes are handled by `squid.h`

```suggestion
```